include/ginkgo/core/log/batch_logger.hpp

```diff
@@ -101,7 +103,7 @@ class BatchConvergence final : public gko::log::Logger {
     using mask_type = gko::log::Logger::mask_type;
 
     void on_batch_solver_completed(
-        const array<int>& iteration_count,
+        const array<int64>& iteration_count,
```

Comment:
Is it the bug of logger or misused in the kernel?
we can only change the public interface if it is the first case.

Reply:
I think the logger needs to store int64 to be efficient. I think we can right now safely assume that no one derives from the BatchConvergence logger.